integration/integration_test.go

```diff
@@ -221,6 +231,125 @@ func call[Resp any](module, verb string, req obj, onResponse func(t testing.TB,
 	}
 }
 
+func setUpKotlinModuleDb(dir string) assertion {
+	return func(t testing.TB, ic itContext) error {
+		output, err := exec.Capture(ic, dir, "docker", "ps",
+			"-a",
+			"--filter", fmt.Sprintf("name=^/%s$", "test-db-1"),
+			"--format", "{{.Names}}")
+		assert.NoError(t, err)
+
+		// provision external DB
+		if len(output) == 0 {
+			err := exec.Command(ic, log.Debug, dir, "docker", "run",
+				"-d",
+				"--rm",
+				"--name", "test-db-1",
+				"--user", "postgres",
+				"-e", "POSTGRES_PASSWORD=secret",
+				"-p", "54321:5432",
+				"postgres:latest", "postgres",
+			).Run()
+			assert.NoError(t, err)
+		} else {
+			// Start the existing container
+			_, err = exec.Capture(ic, dir, "docker", "start", "test-db-1")
+			if err != nil {
+				return err
+			}
+		}
+
+		// write db.yaml file in module
+		dbYaml := fmt.Sprintf(`databases:
+  my-db:
+    jdbcUrl: jdbc:postgresql://%s:5432/postgres
+    username: postgres
+    password: secret
+`, getHostAddr(t, ic))
+		err = os.MkdirAll(filepath.Join(dir, "src/main/resources"), 0700)
+		assert.NoError(t, err)
+
+		err = writeFile(t, filepath.Join(dir, "src/main/resources/db.yaml"), dbYaml)
+		assert.NoError(t, err)
+
+		// add DbTest.kt with a new verb that uses the db
+		content, err := os.ReadFile(filepath.Join(ic.rootDir, "integration/testdata/database/DbTest.txt"))
```

Comment:
It would be simpler to use [scaffolder](https://github.com/TBD54566975/scaffolder) for this. Just pre-recreate the module under `testdata` somewhere, then scaffold it straight into the destination.